packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/perl/changeClassName2.yml

```diff
@@ -0,0 +1,46 @@
+languageId: perl
```

Comment:
Worth adding a test from within the body of both classes

Reply:
From the top body it doesn't work yet (which is technically correct; the nodes following the package and name nodes are siblings, not children, and thus can't be found). I've added a test for the second one, but the first one will need matcher improvement in some way.

Reply:
should be able to just do something like `source_file.package_statement.package_name!`